packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -59,10 +59,10 @@
         <li v-if="hasPagination" class="files-view-options-list-item">
           <oc-page-size
             v-if="!queryParamsLoading"
-            :selected="queryItemAsString(itemsPerPageCurrent)"
+            :selected="itemsPerPageCurrent"
```

Comment:
That was actually fine:

```suggestion
            :selected="queryItemAsString(itemsPerPageCurrent)"
```

Reply:
wtf